### DISCUSSION TITLE
Run vs  Run Anime  vs  Run Realistic

### DISCUSSION THREAD

**Discussion by persius77** [lllyasviel/Fooocus#2732] [Q&A] [answered]
I was curious to know what is the difference between the three versions of the .bat files that Fooocus uses?   I've been looking for literature online but there doesn't seem to be very much on this subject. 

Any thoughts? 
Thanks.

**Comment by Rogala** [comment, marked as answer]
No difference at the moment, there used to be no presets, when they added them to the interface it became easier to switch between them and not have to run different *.bat.

**Comment by persius77** [comment]
Thanks for letting me know.  I was watching some youtube videos and many
people say, "Always use "Run_Realistic for best results".    All three seem
the same to me.

**Reply by mashb1t** [reply]
The preset is the only thing which changes, but you can simply change it in the UI now. Please keep in mind that also the "initial" preset changes depending on which .bat file you've used to start Fooocus with as well as when reloading the page, this initial preset is reapplied again.

**Reply by persius77** [reply]
Thanks for answering.   Much appreciated.
